mobile/src/navigation/app/components/drawer-item/drawer-item.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigation } from '@react-navigation/native';
+import React, { FC } from 'react';
+import { Pressable } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import { AppColor } from '~/common/enums/enums';
+import { DrawerNavigationItem } from '~/common/types/types';
+import { Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+const CustomDrawerItem: FC<DrawerNavigationItem> = ({
+  name,
+  icon,
+  focusedRouteName,
+}) => {
+  const navigation = useNavigation();
+  const focused = focusedRouteName === name ? true : false;
+  const color = focused ? AppColor.TEXT.GRAY_100 : AppColor.TEXT.GRAY_200;
+  const backgroundColor = focused ? AppColor.BRAND.BLUE_100 : 'transparent';
+
+  const handleNavigateTo = (name: any): void => {
```

Comment:
```suggestion
  const handleNavigateTo = (name: AppScreenName): void => {
```

Reply:
Use navigation object from higher level, from `CustomDrawerContent`, not from `useNavigation` hook

Reply:
And use `navigation` object from higher level, from CustomDrawerContent, not from useNavigation hook